internal/router/router.go

```diff
@@ -59,32 +46,6 @@ func ConfiguredRouter(
 	return router, nil
 }
 
-func configureLegacyRoutes(
-	router *Router,
-	server *rpc.Server,
-	supportedChainIDs []tableland.ChainID,
-	rateLim mux.MiddlewareFunc,
-	ctrl *controllers.Controller,
-) {
-	router.post("/rpc", func(rw http.ResponseWriter, r *http.Request) {
-		server.ServeHTTP(rw, r)
-	}, middlewares.WithLogging, middlewares.OtelHTTP("rpc"), middlewares.Authentication, rateLim)
-
-	// Gateway configuration.
-	router.get("/chain/{chainId}/tables/{tableId}", ctrl.GetTable, middlewares.WithLogging, middlewares.OtelHTTP("GetTable"), middlewares.RESTChainID(supportedChainIDs), rateLim)                                        // nolint
-	router.get("/chain/{chainId}/tables/{id}/{key}/{value}", ctrl.GetTableRow, middlewares.WithLogging, middlewares.OtelHTTP("GetTableRow"), middlewares.RESTChainID(supportedChainIDs), rateLim)                         // nolint
-	router.get("/chain/{chainId}/tables/controller/{address}", ctrl.GetTablesByController, middlewares.WithLogging, middlewares.OtelHTTP("GetTablesByController"), middlewares.RESTChainID(supportedChainIDs), rateLim)   // nolint
-	router.get("/chain/{chainId}/tables/structure/{hash}", ctrl.GetTablesByStructureHash, middlewares.WithLogging, middlewares.OtelHTTP("GetTablesByStructureHash"), middlewares.RESTChainID(supportedChainIDs), rateLim) // nolint
-	router.get("/schema/{table_name}", ctrl.GetSchemaByTableName, middlewares.WithLogging, middlewares.OtelHTTP("GetSchemaFromTableName"), rateLim)                                                                       // nolint
-
-	router.get("/query", ctrl.GetTableQuery, middlewares.WithLogging, middlewares.OtelHTTP("GetTableQuery"), rateLim) // nolint
-	router.get("/version", ctrl.Version, middlewares.WithLogging, middlewares.OtelHTTP("Version"), rateLim)           // nolint
-
-	// Health endpoint configuration.
-	router.get("/healthz", controllers.HealthHandler)
-	router.get("/health", controllers.HealthHandler)
-}
```

Comment:
I removed all the legacy routes here. Double check that we actually want to get rid of all of these?